azure-pipelines.yml

```diff
@@ -151,10 +151,11 @@ stages:
   - ${{ if eq(variables._RunAsInternal, True) }}:
     - template: /eng/common/templates/job/onelocbuild.yml
       parameters:
+        UseCheckedInLocProjectJson: true
```

Comment:
Could you do a rebase instead of a merge? Looks like some files have incorrect changes.